templates/openwisp2/settings.py

```diff
@@ -332,7 +387,7 @@
 # HTML minification with django pipeline
 PIPELINE = { 'PIPELINE_ENABLED': True }
 # static files minification and invalidation with django-compress-staticfiles
-STATICFILES_STORAGE = 'compress_staticfiles.storage.CompressStaticFilesStorage'
+STATICFILES_STORAGE = 'openwisp2.storage.OpenwispCompressStaticFilesStorage'
```

Comment:
This was giving an error for openwisp-radius! 😄

Reply:
@atb00ker this is occurring becausue https://github.com/openwisp/openwisp-radius/commit/0819e35bef389b56b12fbba0ab85d9a679020106 was added after version 0.1.0. This should not occur if you test with the latest master. I guess we will require issuing 0.2.0 release of openwisp-radius.